data/README.md

```diff
@@ -0,0 +1,107 @@
+# Dataset Documentation
+
+Details for each field can be found in the [datapackage.yaml](datapackage.yaml)
```

Comment:
fwiw, pretty sure these links won't work in the deposited version of the readme

my inclination is to take them out so we don't forget

Reply:
10f892279a5e36c37e8115de9a4bf1aea95b5992 ✅